eyeball-im-util/src/vector/filter.rs

```diff
@@ -21,15 +27,22 @@ pin_project! {
     }
 }
 
-impl<S, T, F> Filter<S, F>
+impl<S, F> Filter<S, F>
```

Comment:
I would have loved to keep the `T` parameter instead of `VectorDiffContainerStreamElement<S>`, but the Rust compiler was not happy about that (unconstrained generic parameter error).